applications/asset_tracker_v2/src/modules/data_module.c

```diff
@@ -680,8 +680,12 @@ static void requested_data_list_set(enum app_module_data_type *data_list,
 static void on_cloud_state_disconnected(struct data_msg_data *msg)
```

Comment:
Why is this in a method called on_cloud_state_**disconnected**?

Reply:
The event that triggers the ```config_send()``` function for this particular case is ```CLOUD_EVT_CONNECTED```. This event is handled when the application is in an **disconnected** state. However, this can be ignored. The scope of this PR has changed.